twoliter/embedded/Makefile.toml

```diff
@@ -464,19 +461,6 @@ rc=0
 export VARIANT="${BUILDSYS_VARIANT}"
 
 # For rust first-party source code
-if ! docker run --rm \
-   -u $(id -u):$(id -g) \
-   -e CARGO_HOME="/tmp/.cargo" \
-   -v "${CARGO_HOME}":/tmp/.cargo \
-   -v "${BUILDSYS_ROOT_DIR}/sources":/tmp/sources \
-   -v "${BUILDSYS_ROOT_DIR}/tools":/tmp/tools \
-   "${BUILDSYS_SDK_IMAGE}" \
-   cargo clippy \
-  --manifest-path /tmp/tools/Cargo.toml \
-  --locked -- -D warnings --no-deps; then
```

Comment:
The `check-shell` test was previously linting some of the migrated scripts so we should bring that back to this repo at some point.

Reply:
> The `check-shell` test was previously linting some of the migrated scripts so we should bring that back to this repo at some point.

I have an issue open for this https://github.com/bottlerocket-os/twoliter/issues/42